### PR TITLE
Update code for directory checking

### PR DESCRIPTION
**Pull request by LexLorenzo**
This gets all the cases:
image

This divides the cases according to their cohorts:
image

### CODE REVIEW

File: final_project.ipynb [hunk 1]
```diff
@@ -13,6 +13,164 @@
...
+     },
+     "execution_count": 17,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "import os\n",
+    "\n",
+    "testing_label_relative = 'Thesis_Folder\\VALDO_Dataset\\Task2'\n",
```
Comment (JoshuaEntrata) [inline final_project.ipynb:116]: use environment variable

File: final_project.ipynb [hunk 2]
```diff
@@ -119,7 +286,7 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 5,
    "metadata": {},
    "outputs": [],
    "source": [
```
Comment (JoshuaEntrata) [inline final_project.ipynb:292]: for `device_num`, assign this instead para applicable sa different devices

`device_num = torch.cuda.get_device_name(0)`

**Review by JoshuaEntrata — COMMENTED** [review]
adfsfasd

**Review by 1123Alessandro — CHANGES_REQUESTED** [review]
Remove logs

**Comment by LexLorenzo** [comment]
> Remove logs

Done

**Review by 1123Alessandro — APPROVED** [review]

**Pull request merged**